devops/clean

```diff
@@ -36,6 +36,11 @@ function remove_unwanted_files() {
         install_files/ansible-base/tor_v3_keys.json \
         build/*.deb
 
+    # Remove any Onion URL from make dev-tor
+    if docker volume sd-onion-services; then
```

Comment:
Should this be `docker volume inspect sd-onion-services`? This is incorrect syntax (at least on docker, so it will never return 0, and the volume will never be nuked.